app/Jobs/GetJobData.php

```diff
@@ -35,11 +40,16 @@ public function handle(): void
             ApiKeyNotFoundException::validateApiKey($apiKey);
 
             $categories = JobCategory::all();
+
```

Comment:
You should move this logic to your CategoryNotFoundException class like what I do in ApiKeyNotFoundException class

Reply:
Thank you for the suggestion! I’ve moved the logic to the `CategoryNotFoundException` class, similar to how you structured the `ApiKeyNotFoundException` class. Let me know if there’s anything else I should adjust.